tests/pytests/unit/modules/test_suse_ip.py

```diff
@@ -0,0 +1,697 @@
+"""
+    :codeauthor: Jayesh Kariya <jayeshk@saltstack.com>
+"""
+
```

Comment:
```suggestion
```
Let's remove this docstring, Jayesh was not the author of these SUSE specific tests.